### ISSUE TITLE
There is still uncertainty to get old icons back

### ISSUE THREAD

**Issue by Ransom77** [DrunkenCommie/LibreOffice-old-extension-icons#1]
Hi DrunkenCommie,

As a Boomer myself, I’m really glad to have found your site because, like you, the new file extension icons (but also the new branding icons) of LibreOffice really bother me.

But what I did not understand: How do I tell my Operating System (Win10 Pro) to use the old icons (downloaded from your site) instead of the new ones? Are there any other ways without using FilesTypeMan?

Nevertheless, I have tried FilesTypeMan once in a VM, and it seems to work, but I am not sure which symbols to choose exactly, because for example for .123 I don’t know if I should use the spreadsheet-old.ico, the oasis-spreadsheet-old.ico or the calc_app-old.ico. In general, I don’t really understand how FilesTypeMan works. For example, do the paths for the replaced (old) icons always have to remain “stable”, i.e. in the sense that they must never change?

And: Is there any way to replace the old branding icons in the actual program files (sbase.exe, scal.exe, sdraw.exe, simpress.exe, smath.exe, soffice.exe and swriter.exe)? If you open these exe files with e.g. WinZip or 7-Zip, it is not possible to replace the new branding symbols with the old ones, which I had hoped to do through Mike Kaganski’s advice.

I have extracted the old and the new branding symbols and created a zip file similar to your zip file with the old an the new extension icons.

Greetings,
Ransom77
[LibreOffice-branding-icons-main.zip](https://github.com/DrunkenCommie/LibreOffice-old-extension-icons/files/15500729/LibreOffice-branding-icons-main.zip)

**Comment by Ransom77** [comment]
Now I have found a good, simple and practicable solution to be able to continue running LO installations ≥ version 7.5 with the old icon set (since version 6.1). (However, creating this solution and researching it took me two days).

I tried the FileTypesMan tool mentioned above and discovered that the registry always refers to the same file regarding the default icons, namely soffice.bin. So I thought to myself, if I just save the soffice.bin from the last LO suite (7.4.7.2) (see file attachment – change the file extension from *.png to *.bin), copy it to a different folder than the already existing soffice.bin of the current installation and change the registry entries accordingly, I will get the old icons in a new installation as well. This assumption was confirmed.

So I laboriously copied out all 197 registry entries that refer to the default icon of LO and created a reg file that I can easily use for future installations (see file attachment – change the file extension from *.txt to *.reg). I changed the original path of each entry (@=“C:\\Program Files\\LibreOffice\\program\\...”) to @=“C:\\ProgramData\\LibreOffice\\program\\...”.

For a new installation, proceed as follows: First install the new version of LO, but do not open a program, but restart. After the restart, first create a folder “LibreOffice” under C:\ProgramData and then a folder “program” underneath. Copy the saved soffice.bin from the last version with the old icons into this folder. Now double-click the created reg file, restart the program and the old icons are back in the shortcuts.

Incidentally, it does not matter whether you use a soffice.bin (v. 7.4.7.2) from a 32-bit or 64-bit installation. Both files work with both 32-bit and 64-bit installations.

![soffice](https://github.com/DrunkenCommie/LibreOffice-old-extension-icons/assets/66835201/4f7d90d4-994a-4c3a-a602-faf106bb5da1)

[LibreOffice_DefaultIcons.txt](https://github.com/user-attachments/files/15522358/LibreOffice_DefaultIcons.txt)
